Trap Quest.inform/Inform/Extensions/Sex Toys/Ben Wa Balls.i7x

```diff
@@ -0,0 +1,43 @@
+Ben Wa Balls by Sex Toys begins here.
+
+A ben wa balls is a kind of plug. The printed name of ben wa balls is usually "[TQlink of item described][item style][unless magic-curse of the item described is bland or curse-ID of the item described is unsure][magic-curse] [end if]ben wa balls[clothing-title-after][TQxlink of item described][verb-desc of item described]". The printed plural name of ben wa balls is usually "[TQlink of item described][clothing-title-before]sets of ben wa balls[clothing-title-after][TQxlink of item described][verb-desc of item described]". The text-shortcut of ben wa balls is "bwb". There is 1 ben wa balls.
+
+Figure of ben wa balls is the file "benWaBalls1.png".
+
+To decide which figure-name is clothing-image of (C - ben wa balls):
+	decide on figure of ben wa balls.
+
+To say ClothingDesc of (C - ben wa balls):
+	say "Two balls [if the item described is cursed and the curse-ID of the item described is sure]of plastic on a string. One pops out from time to time, embarassingly, but the other won't come out due to its curse[otherwise]made of metal[end if][if the item described is worn]. Their turning, when you move, slowly builds your arousal.[otherwise].[end if]".
```

Comment:
You don't want to use `item described` here ideally, you want to use `C`. If you copied this from another item please tell me which one as that should be changed too. If ClothingDesc was ever called by any other means other than accessing the description of the item, it would behave incorrectly.